src/index.css

```diff
@@ -1,23 +1,57 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Mochiy+Pop+One&family=Roboto+Slab:wght@300;400;500;600;900&family=Source+Code+Pro:wght@400;500;600;700&display=swap');
+
 * {
   margin: 0;
   box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  font-family: 'Mochiy Pop One', sans-serif;
+  font-family: 'Roboto Slab', serif;
+  font-family: 'Source Code Pro', monospace;
+}
+
+body {
+  background-color: #fafafa;
 }
 
```

Comment:
- [ ] You want to consider using only 2 font families that are: ✍️ 
- RobotoSlab
- Montserrat

Currently, your project is using another one. Please, check the template again to notice which text is using Roboto and which is using Montserrat.

![font-bookstore](https://user-images.githubusercontent.com/79876959/145661278-fd59d64b-b3a3-481b-b31f-bbeaafcc511b.PNG)